.sauce/testcafe.yml

```diff
@@ -1,18 +1,37 @@
 apiVersion: v1alpha
-metadata:
-  name: Testing TestCafe Support
-  tags:
-    - e2e
-    - release team
-    - other tag
-  build: Release $CI_COMMIT_SHORT_SHA
-files:
-  - ./tests/e2e/testcafe/example.test.js
-suites:
-  - name: "saucy test"
-    match: ".*.(spec|test).[jt]s$"
-image:
-  base: saucelabs/stt-testcafe-node
-  version: latest
+kind: testcafe
 sauce:
   region: us-west-1
+  concurrency: 1 # Controls how many suites are executed at the same time (sauce test env only).
+  metadata:
+    name: Testing Testcafe Support
+    tags:
+      - e2e
+      - release team
+      - other tag
+    build: Release $CI_COMMIT_SHORT_SHA
+suites:
+  - name: "saucy test"
+    browser: "firefox"
+    src:
+      - ".*.(spec|test).[jt]s$" # test files glob
```

Comment:
That's not a glob though but a regex :p Would be good to put the default glob example of testcafe in here

Reply:
sure!